EventProc/python/overlay.py

```diff
@@ -0,0 +1,38 @@
+"""Configuration for overlay """
+
+
+
+from LDMX.Framework import ldmxcfg
+
+class OverlayProducer(ldmxcfg.Producer) :
+    """Configuration for pileup overlay
+
+        Sets all parameters to reasonable defaults.
+
+    Examples
+    --------
+        from LDMX.EventProc.overlay import OverlayProducer
+        p.sequence.append( OverlayProducer() )
+    """
+
+    def __init__(self,name = 'overlay') :
```

Comment:
The constructor is the perfect place for all the "required" variables. From my perspective, I would add the overlay file name here so that a user can't construct the producer without specifying a file. e.g.
```
def __init__(self,file_name) : 
        super().__init__(name,'ldmx::OverlayProducer','EventProc')
        self.overlayFileName = file_name
```

Reply:
Good suggestion, taken.